server/asset/common.go

```diff
@@ -21,6 +21,9 @@ type Backend interface {
 	InitTxSize() uint32
 	// CheckAddress checks that the given address is parseable.
 	CheckAddress(string) bool
+	// CoinIDString attempts to decode the coinID, returning an asset-specific
+	// string representation of the coinID.
+	CoinIDString(coinID []byte) (string, error)
```

Comment:
It's not quite the same thing as a hexadecimal conversion from the `[]byte` since only the asset backend can decode it.  Note the dcr and btc implementations currently form "{hex}:{vout}", and only the backend knows which of the `[]byte`s become hex and which are vout.  The main idea of `CoinIDString` is to provide meaningful logging.  I ran into cases where a string from Coin.ID.String was hard to understand, and that was client side.

Reply:
I meant to add a `Stringer` to `asset.Coin`, not `order.CoinID`.